pages/integrations/inputs/o365_input.rst

```diff
@@ -0,0 +1,28 @@
+.. _o365_input:
+
+*********************
+Office 365 Integrations
+*********************
+
+.. attention:: This is a Graylog Enterprise feature and is only available since Graylog version 3.3. A valid Graylog Enterprise license is required.
+
+
+Office 365 Integrations plugin, collects data from Windows Logs, Email, File Sharing and Data Loss Prevention Logs in one input.
```

Comment:
We might need to add here that this is for the hosted "online" office and not to collect logs from a windows server that is in your datacenter.

Reply:
@jalogisch - what is a hosted "online" office ? you mean to say that they use all the microsoft office applications in the cloud .

Reply:
@jalogisch - I added your feedback in the attention section. please validate.